.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this is an auto-generated file right? if so (and there is no good reason to keep it), please remove it and add it to `.gitignore` file